src/BlazingPizza.Client/JSRuntimeExtensions.cs

```diff
@@ -1,7 +1,4 @@
 ﻿using Microsoft.JSInterop;
-using System;
-using System.Collections.Generic;
-using System.Linq;
```

Comment:
If you're really interested in doing all of this namespace cleanup, then it should be done for all of the various save points as well, I don't want the *final* files to get out of sync with the save points.

Reply:
This makes sense. While cleaning up, I found some saving points projects when I could also replace OrderState.Order with Order.